.azure-pipelines/linux-conda-CI.yml

```diff
@@ -14,27 +14,35 @@ jobs:
   strategy:
     matrix:
       # No python 2.x since tf2onnx does not support it
-      Python35-tf1110:
```

Comment:
Any problem with Python 3.5?

Reply:
Fail to create Anaconda Env in Linux build today.